README.md

```diff
@@ -68,26 +68,15 @@ client = genai.Client()
 
 By default, the SDK uses the beta API endpoints provided by Google to support
 preview features in the APIs. The stable API endpoints can be selected by
-setting the API version to `v1`.
+setting the API version to `v1`. The alpah API endpints can be selected by
+setting the API version to `v1alpha`.
 
-To set the API version use `http_options`. For example, to set the API version
-to `v1` for Vertex AI:
-
-```python
-client = genai.Client(
-    vertexai=True,
-    project='your-project-id',
-    location='us-central1',
-    http_options=types.HttpOptions(api_version='v1')
```

Comment:
I think this is intended because we want to demonstrate how to use GA version of the API for Vertex.